srcpkgs/nsxiv/patches/no-rebuild-options.patch

```diff
@@ -1,7 +1,7 @@
 Preventing rebuild of options.o when installing
 --- a/Makefile
 +++ b/Makefile
-@@ -106,7 +106,7 @@
+@@ -81,7 +81,7 @@
  		rm -f "$$dir/nsxiv.png"; \
  	done
 
```

Comment:
this patch refresh should not be necessary

Reply:
But the patch offsets -25 lines. Won't that be a problem?

Reply:
no